test/unit/communication/test_broadcast_panel.cpp

```diff
@@ -98,25 +94,37 @@ TYPED_TEST(PanelBcastTest, BroadcastRow) {
       testBroadcast<TypeParam, Coord::Row>(cfg, comm_grid);
 }
 
+TYPED_TEST(PanelBcastTest, BroadcastColStoreTransposed) {
+  for (auto comm_grid : this->commGrids())
+    for (const auto& cfg : test_params)
+      testBroadcast<TypeParam, Coord::Col, StoreTransposed::Yes>(cfg, comm_grid);
+}
+
+TYPED_TEST(PanelBcastTest, BroadcastRowStoreTransposed) {
+  for (auto comm_grid : this->commGrids())
+    for (const auto& cfg : test_params)
+      testBroadcast<TypeParam, Coord::Row, StoreTransposed::Yes>(cfg, comm_grid);
+}
+
 std::vector<config_t> test_params_bcast_transpose{
     {{0, 0}, {1, 1}, {0, 0}},    // empty matrix
     {{10, 10}, {2, 2}, {5, 5}},  // empty panel (due to offset)
     {{20, 20}, {2, 2}, {9, 9}},  // just last tile (communicate without transpose)
     {{25, 25}, {5, 5}, {1, 1}},
 };
 
-template <class TypeParam, Coord PANEL_SRC_AXIS>
+template <class TypeParam, Coord AxisSrc, StoreTransposed storageT = StoreTransposed::No>
```

Comment:
Maybe in the test `storageT` can be explicit (i.e. not defaulted)?

Reply:
The rationale was: I don't want to touch the code, so I'd like to have tests that represents also this aspect.

I agree it is not so "transparent". Considering also #725, would you prefer to have the storage parameter explicit also in the code?

Reply:
So I think it's ok to have it be defaulted on the `Panel` class itself. To me it's clearer if it's explicit in the test because then I can see for sure that both cases are being tested, but it's really not a big issue. I'm happy the way it is as well.